tests/test_smallfile.sh

```diff
@@ -4,27 +4,20 @@ set -xeEo pipefail
 source tests/common.sh
 
 function finish {
-  if [ $? -eq 1 ] && [ $ERRORED != "true" ]
-  then
-    error
-  fi
-
-  echo "Cleaning up Smallfile"
-  kubectl delete -f tests/test_crs/valid_smallfile.yaml
-  delete_operator
+  kubectl delete all --all -n my-ripsaw
```

Comment:
same comment as with test_fiod.   Why was the delete_operator function a bad idea?

Reply:
You're right. It's easy to fix, so let's include the code change here.

Reply:
Same comment re wait_clean as before. Also, why are you removing the error if statement?